deploy/charts/trust-manager/templates/rbac-per-namespace.yaml

```
@@ -0,0 +1,51 @@
+{{- if .Values.app.targetNamespaces }}
+{{- $targetNamespaces := .Values.app.targetNamespaces -}}
+{{- $allNamespaces := append (append $targetNamespaces .Values.app.trust.namespace) (include "trust-manager.namespace" . | trim) -}}
+{{- range $ns := uniq $allNamespaces }}
```

```suggestion
{{- $namespaces := append (append .Values.app.targetNamespaces .Values.app.trust.namespace) (include "trust-manager.namespace" . ) -}}
{{- range $ns := uniq $namespace }}
```

It's probably also a good idea to deduplicate the `namespaces` variable.